editor/import/3d/resource_importer_scene.cpp

```diff
@@ -42,20 +42,22 @@
 #include "scene/3d/mesh_instance_3d.h"
 #include "scene/3d/navigation/navigation_region_3d.h"
 #include "scene/3d/occluder_instance_3d.h"
+#include "scene/animation/animation_player.h"
+#include "scene/resources/3d/importer_mesh.h"
+#include "scene/resources/animation.h"
+#include "scene/resources/bone_map.h"
+#include "scene/resources/packed_scene.h"
+#include "scene/resources/resource_format_text.h"
+
+// 3D physics.
```

Comment:
In a few cases, I also grouped together physics includes. This can be used with `#ifndef PHYSICS_3D_DISABLED`, but it's also a nice organization on its own, I would argue.